_posts/2025-02-21-release.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Release 2025.09
+author: Priya
+date: 2025-02-21
+category: Jekyll
+layout: post
+---
+### Product Enhancement :
+##### Automation-
+Mammoth introduces new features to enhance the automation creation process. Users are now required to define at least one action and condition, each with configurable details. Multiple actions can be added to an automation for greater flexibility.
```

Comment:
New Alpha Features: Automation & Dataset Schema Editing

We're introducing two new features to enhance automation and dataset management in Mammoth. These features are currently in alpha and available on request.

Automation: Users can now automate manual file uploads into existing datasets. The new schema mapping customization in automation configuration allows users to map multiple incoming columns to a single destination column for greater flexibility.

Dataset Schema Editor: This feature enables users to edit dataset schemas effortlessly through an intuitive interface, providing better control over schema configurations.

These features are now live as alpha releases. If you're interested in early access, reach out to us at support@mammoth.io or via the in-app chat.